pkgs/development/python-modules/seqnmf/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, matplotlib
+, scipy
+, seaborn
+}:
+
+buildPythonPackage rec {
+  pname = "seqnmf";
+  version = "0.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "75a7bdb5f23de6345625f0d36bb0e42399f270ebe0d411d5a6ee64f6e7325f51";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    matplotlib
+    scipy
+    seaborn
+  ];
+
+  # no tests
+  doCheck=false;
```

Comment:
looking at https://github.com/ejolly/seqnmf, it looks like they don't have any tests, and haven't had an update in 18+ months :(

Reply:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.